proposals/003-credentials.md

```diff
@@ -24,113 +24,30 @@ My requirements are:
  * Have Syndesis manage credential lifecycle
 
-## Discussion 
+## Discussion
 
 The idea of credential provider is to encapsulate managing specifics of 3rd party SaaS credentials.
 
 Currently credentials are represented by connection properties that are marked as secret. This proposes an abstraction over existing connection properties with the ability to implement specific functionality for getting credentials such as OAuth tokens from 3rd party SaaS solutions. As such existing connection properties and their persistence as k8s secrets would remain the same, the UI would also have the choice of either managing the connection properties in the same way as it is currently done or can call into credentials provider implementation, specific to each SaaS solution, that would *apply* credentials to connection properties on the backend.
-Each 3rd party SaaS can have slightly different OAuth flow which mandates the need for credential provider, a plug in, possibly shipped as a separate artifact and discovered at runtime, mounted at `/api/{version}/credentials/{connector}` to facilitate the OAuth flow and manage (i.e. *apply*) the resulting credentials.
-Considering that OAuth requires per-user and per-application secrets, there is a need for system (Syndesis owned) and user (citizen user owned) credentials. Melding of those would be also responsibility of credential provider.
+Each 3rd party SaaS can have slightly different OAuth flow which mandates the need for credential provider, a plug in, possibly shipped as a separate artifact and discovered at runtime.
+Considering that OAuth requires per-user (OAuth token) and per-application secrets (Client Secret), there is a need for system (Syndesis owned) and user (citizen user owned) credentials. Melding of those would be also responsibility of credential provider.
 
-Management of user and system credentials is also performed by credential provider, for instance initial setup of system credentials or refresh of user credentials.
+Management of user and system credentials is also performed by credential provider, for instance initial setup of system credentials or refresh of user credentials (where applicable).
 
 
 ## User interaction
 
-A citizen user during the creation of new connection selects `conect to _Service_`, this leads the user to the 3rd party service requiring the users credential to authorise Syndesis access on the users behalf. The user authenticates and authorizes Syndesis's access, and is returned to the same create connection screen which now shows that Syndesis is allowed to access the service.
+A citizen user during the creation of new connection selects `conect to _Service_`, this leads the user to the 3rd party service requiring the users credential to authorize Syndesis access on the users behalf. The user authenticates and authorizes Syndesis's access, and is returned to the same create connection screen which now shows that Syndesis is allowed to access the service.
```

Comment:
btw authorize and authorise are both fine in the land of real English :stuck_out_tongue: